Extensibility/TemplatePack/ProjectTemplates/CSharp/Forms/PrismUnityApp/UnityApp.Droid/Resources/values/styles.xml

```diff
@@ -7,5 +8,11 @@
     <item name="colorAccent">@color/accent</item>
     <item name="android:windowBackground">@color/window_background</item>
   </style>
+  <style name="Theme.Splash" parent="android:Theme">
+    <item name="android:windowBackground">@layout/SplashScreen</item>
+    <item name="android:windowNoTitle">true</item>
+    <item name="android:windowIsTranslucent">false</item>
+    <item name="android:windowIsFloating">false</item>
+    <item name="android:backgroundDimEnabled">true</item>
+  </style>
```

Comment:
I'd probably add `<item name="android:windowFullscreen">true</item>` in commented form in case anyone wants to hide the status bar. For beginners, finding this syntax could be a bit of work.

Also, the following is probably unnecessary?
 +    <item name="android:windowIsTranslucent">false</item>
 +    <item name="android:windowIsFloating">false</item>
 +    <item name="android:backgroundDimEnabled">true</item>